components/login/SignInArea.tsx

```diff
@@ -26,6 +28,15 @@ const SignInArea = () => {
     }
   };
 
+  useEffect(() => {
+    const error = githubError || googleError;
+
+    if (!error) return;
+    toast.error(
+      authErrorCodes[error.code as keyof typeof authErrorCodes] || error.message
+    );
+  }, [githubError, googleError]);
+
```

Comment:
While this might work, I'm a bit unsure about using useEffect for showing toasts.

The docs mention that the useSignInWith() functions return undefined if there's an error. I think we should use those.
https://github.com/CSFrequency/react-firebase-hooks/tree/master/auth

Reply:
We could use a modal that shows when the errors exists too right?

Reply:
> We could use a modal that shows when the errors exists too right?

Don't need a modal, toast would be good enough.

Reply:
They don't trigger like we want as far as I tested... they only show the toast on triggering of login i.e. when we click either of the sign in buttons